core/src/main/java/apoc/nodes/Nodes.java

```diff
@@ -69,15 +69,18 @@
     public Pools pools;
 
     @Procedure(mode = Mode.WRITE)
-    @Description("apoc.nodes.link([nodes],'REL_TYPE') - creates a linked list of nodes from first to last")
-    public void link(@Name("nodes") List<Node> nodes, @Name("type") String type) {
+    @Description("apoc.nodes.link([nodes],'REL_TYPE',checkExistence=false) - creates a linked list of nodes from first to last. If 'checkExistence=true' it does not create the relationship if it already exists")
+    public void link(@Name("nodes") List<Node> nodes, @Name("type") String type, @Name(value = "checkExistence", defaultValue = "false") boolean checkExistence) {
```

Comment:
no fallo come configuration param 
```suggestion
    public void link(@Name("nodes") List<Node> nodes, @Name("type") String type, @Name(value = "config", defaultValue = "{}") Map<String, Object> config) {
```

Reply:
inoltre chiama la proprietà `avoidDuplicates` default false come già hai fatto